.github/workflows/basic.yml

```diff
@@ -17,14 +17,15 @@ jobs:
         pool_tracking: ['ON', 'OFF']
         shared_library: ['OFF']
         os_provider: ['ON']
+        sanitizers: ['ON', 'OFF']
```

Comment:
Please set this to off by default and add a single build to "include" list where this is set to "on". Could you also add an TODO "move sanitizers checks the separate workflow"?

Reply:
Done.